### PR TITLE
Fix evidence id

### PR DESCRIPTION
**Pull request by bingen**

### CODE REVIEW

File: src/Arbitrable.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Evidence } from '../types/schema'
 import { EvidenceSubmitted } from '../types/templates/Arbitrable/Arbitrable'
 
 export function handleEvidenceSubmitted(event: EvidenceSubmitted): void {
-  let id = event.transaction.hash.toHex() + event.logIndex.toHex()
+  let id = event.transaction.hash.toHex() + event.logIndex.toHex().slice(2).padStart(4, '0')
```
Comment (sohkai) [inline src/Arbitrable.ts]: I was wondering about this; would it make sense to add a separator like `-` or `:` so it's easier for frontends to split this into the tx hash and log index if they have to?

Comment (facuspagnuolo) [reply src/Arbitrable.ts]: What's the reason for this fix? Should we do it in the rest of the places we are generating an id using the tx hash and the log index?

Comment (facuspagnuolo) [reply src/Arbitrable.ts]: What's the reason for this fix? Should we do it in the rest of the places we are generating an id using the tx hash and the log index?

Comment (facuspagnuolo) [inline src/Arbitrable.ts]: What's the reason for this fix? Should we do it in the rest of the places we are generating an id using the tx hash and the log index?

Comment (bingen) [reply src/Arbitrable.ts]: @fabriziovigevani asked for it, maybe him or @rperez89 can provide more context. It seems there was an ordering issue in the frontend (which I don't fully understand)

Comment (bingen) [reply src/Arbitrable.ts]: Well, the way it is now, it should be easy with slice, as both parts are fixed length.

Comment (fabriziovigevani) [reply src/Arbitrable.ts]: Yes, the reason is that we need to have another field to order the evidences by, in the case that they were created at the same time.

Comment (facuspagnuolo) [reply src/Arbitrable.ts]: @fabriziovigevani 👇 
> Should we do it in the rest of the places we are generating an id using the tx hash and the log index?

Comment (fabriziovigevani) [reply src/Arbitrable.ts]: We have concluded in the standup that we can just slice the tx hash in the frontend and use the resulting `logIndex` for comparison. So this change is not needed.

Comment (bingen) [reply src/Arbitrable.ts]: Ok, closing it then

**Pull request closed without merging**